src/main/java/com/cinema/validation/PasswordValidator.java

```diff
@@ -0,0 +1,21 @@
+package com.cinema.validation;
+
+import com.cinema.model.dto.UserRequestDto;
+import java.util.Objects;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class PasswordValidator implements
+        ConstraintValidator<PasswordConstraint, UserRequestDto> {
+
+    @Override
+    public void initialize(PasswordConstraint constraintAnnotation) {
+    }
+
+    @Override
+    public boolean isValid(UserRequestDto userRequestDto,
+                           ConstraintValidatorContext validatorContext) {
+        return userRequestDto.getPassword() != null && userRequestDto.getRepeatPassword() != null
```

Comment:
```suggestion
        return userRequestDto.getPassword() != null
```
second field check for null is redundant, if this field is null - it's false right away in other case Objects.equals will do all the work